blox/profile.cue

```diff
@@ -11,15 +11,18 @@
 	#Website: {
 		_model: {
 			plural: "websites"
+			supportedExtensions: ["yaml", "yml"]
 		}
 
 		url: string
 		profile_id?: string
+		body?: string
 	}
 
 	#Profile: {
 		_model: {
 			plural: "profiles"
+			supportedExtensions: ["yaml", "yml", "md", "mdx"]
```

Comment:
can you add .svx to all the content types that support markdown please? It's svelte's version of mdx.

Reply:
No

Reply:
Sure, I'll add these extensions to the global model's default